plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -663,7 +667,7 @@ public void close()
     private Iterable<?>[] transformToNulls(Iterable<?>[] values)
     {
         return stream(values)
-                .map(v -> transform(v, constant(null)))
+                .map(v -> Streams.stream(v).map(constant(null)).collect(toList()))
```

Comment:
static import `Streams.stream`

Reply:
The `Arrays#stream` method is already statically imported

Reply:
That's probably why `Streams.stream` has overloads that are otherwise unnecessary -- to allow one static import to rule them all.

(btw, for arrays i personally prefer `Stream.of(args...)`)

Reply:
That's a good suggestion. I didn't know `Stream.of(args...)` exists. Updated.